lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2201,25 +2201,26 @@ SwiftLanguageRuntime::GetRuntimeUnwindPlan(ProcessSP process_sp,
   Address pc;
   pc.SetLoadAddress(regctx->GetPC(), &target);
   SymbolContext sc;
-  if (pc.IsValid()) {
-    pc.CalculateSymbolContext(&sc,
-                              eSymbolContextFunction | eSymbolContextSymbol);
-    if (sc.function) {
-      Address func_start_addr = sc.function->GetAddressRange().GetBaseAddress();
-      AddressRange prologue_range(func_start_addr,
-                                  sc.function->GetPrologueByteSize());
-      if (prologue_range.ContainsLoadAddress(pc, &target) ||
-          func_start_addr == pc) {
-        return UnwindPlanSP();
-      }
-    } else if (sc.symbol) {
-      Address func_start_addr = sc.symbol->GetAddress();
-      AddressRange prologue_range(func_start_addr,
-                                  sc.symbol->GetPrologueByteSize());
-      if (prologue_range.ContainsLoadAddress(pc, &target) ||
-          func_start_addr == pc) {
-        return UnwindPlanSP();
-      }
+  if (pc.IsValid())
+    if (!pc.CalculateSymbolContext(&sc, eSymbolContextFunction |
+                                            eSymbolContextSymbol))
+      return UnwindPlanSP();
+
+  if (sc.function) {
+    Address func_start_addr = sc.function->GetAddressRange().GetBaseAddress();
+    AddressRange prologue_range(func_start_addr,
+                                sc.function->GetPrologueByteSize());
+    if (prologue_range.ContainsLoadAddress(pc, &target) ||
+        func_start_addr == pc) {
```

Comment:
Remove extra `{}` to conform to LLVM coding style.

Reply:
Is there a comment we could add here explaining why this case cannot return an unwind plan?

Reply:
note that this block of code is not new, it's basically unchanged (indention-only change)

Reply:
The change is essentially this:

```diff
@@ -2201,9 +2201,11 @@ SwiftLanguageRuntime::GetRuntimeUnwindPlan(ProcessSP process_sp,
   Address pc;
   pc.SetLoadAddress(regctx->GetPC(), &target);
   SymbolContext sc;
-  if (pc.IsValid()) {
-    pc.CalculateSymbolContext(&sc,
-                              eSymbolContextFunction | eSymbolContextSymbol);
+  if (pc.IsValid())
+    if (!pc.CalculateSymbolContext(&sc, eSymbolContextFunction |
+                                            eSymbolContextSymbol))
+      return UnwindPlanSP();
+
   if (sc.function) {
     Address func_start_addr = sc.function->GetAddressRange().GetBaseAddress();
     AddressRange prologue_range(func_start_addr,
```

Reply:
I'd prefer not to mix in NFC changes with this, unless you feel strongly about it, then I'm 👍

Reply:
Feel free do do this as a follow-up commit!

Reply:
thanks!